Frontend/CO.CDP.OrganisationApp/Pages/Index.cshtml

```diff
@@ -8,6 +8,21 @@
 
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds govuk-body">
+            @if (Request.Query.ContainsKey("one-login-error"))
+            {
+                <div class="govuk-error-summary" data-module="govuk-error-summary">
+                    <div role="alert">
+                        <h2 class="govuk-error-summary__title">
+                            There is a problem with one-login, please retry
```

Comment:
We will need to agree on what message to show here with BAs/Content Designers. I think it would be good to gather all the messages we've defined ourselves so far and put them into a ticket for re-evaluation.

Reply:
Yes, will raise in planning